### ISSUE TITLE
RCT call throws random ConcurrentModificationExceptions

### ISSUE THREAD

**Issue by jdaberkow** [openbase/bco.registry#31]
This exception gets randomly thrown by the call of the following method:
https://github.com/openbase/bco.registry/blob/master/location-registry/remote/src/main/java/org/openbase/bco/registry/location/remote/LocationRegistryRemote.java#L180

```java
FATAL EXCEPTION: pool-6-thread-1
 Process: org.openbase.bco.bcomfy, PID: 24699
 java.util.ConcurrentModificationException
     at java.util.HashMap$HashIterator.remove(HashMap.java:805)
     at rct.impl.TransformerCoreDefault.checkRequests(TransformerCoreDefault.java:675)
     at rct.impl.TransformerCoreDefault.access$000(TransformerCoreDefault.java:26)
     at rct.impl.TransformerCoreDefault$1.run(TransformerCoreDefault.java:232)
     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
     at java.lang.Thread.run(Thread.java:818)
```

**Comment by DivineThreepwood** [comment]
https://github.com/CentralLabFacilities/rct-java/

**Comment by DivineThreepwood** [comment]
Fixed: CentralLabFacilities/rct-java#1 Bug - ConcurrentModificationException during startup phase
RCT switched to master version in jul to verify the fix before performing a new rct release.

**Comment by jdaberkow** [comment]
Thanks. I do not experience this bug anymore. Seems to be fixed.
